apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/MetricsSummary/index.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import { Commit, Evaluation } from '@latitude-data/core/browser'
+import { EvaluationResultWithMetadata } from '@latitude-data/core/repositories'
+import { useCurrentCommit } from '@latitude-data/web-ui'
+
+import { BigNumberPanels } from './BigNumberPanels'
+
+export function MetricsSummary({
+  evaluation,
+  evaluationResults,
+}: {
+  evaluation: Evaluation
+  evaluationResults: EvaluationResultWithMetadata[]
+}) {
+  const { commit } = useCurrentCommit()
+
+  return (
+    <div className='flex gap-6 flex-wrap'>
+      <div className='h-[192px] min-w-[300px] flex-1 bg-muted-foreground rounded-lg' />
+      <div className='h-[192px] min-w-[300px] flex-1 bg-muted-foreground rounded-lg' />
+      <div className='min-w-[400px] flex-1'>
```

Comment:
More hardcoded size

Reply:
In this case this is a placeholder while the actual charts are being developed.